workflows/Teams_Vulnerability_Exception_Handling/workflow.spec.yaml

```
@@ -0,0 +1,23 @@
+extension: workflow
+products: ["insightconnect", "insightvm"]
+name: Teams_Vulnerability_Exception_Handling
+title: "Manage Vulnerability Exception Requests with InsightVM from Microsoft Teams"
+description: "Processes NEW vulnerability exceptions in InsightVM.  Please note, it does NOT trigger on a re-submission of an existing vulnerability exception."
```

To Spencer's point, would recommend:
```suggestion
description: "Receive notifications for and review InsightVM (or Nexpose) vulnerability exception requests from Microsoft Teams."
```

---

Can we please keep the description notifying the user that it does not re-trigger on re-submission?

---

Shouldn't this be in the overview/features and not the description

---

Yeah this should be in the Overview & Usage details. This description field is what is shown on the main extension library page -- the "preview", if you will. This needs to more clearly describe what the workflow does. The way it's written now is a bit discouraging as it focuses more on what the workflow doesn't do than what it does.